Python_Engine/Python/src/python_toolkit/plot/parallel_coordinate_plot.py

```diff
@@ -0,0 +1,138 @@
+import plotly.graph_objects as go
+import pandas as pd
+from matplotlib.colors import Colormap
+import decimal as d
+
+from ..bhom.analytics import bhom_analytics
+
+def set_dimensions(df: pd.DataFrame, tick_mark_count: int, dp:int): -> list[dict[str, any]]
```

Comment:
```suggestion
import decimal as d
from typing import List, Dict

from ..bhom.analytics import bhom_analytics

def set_dimensions(df: pd.DataFrame, tick_mark_count: int, dp:int): -> List[Dict[str, any]]
```

I tend to have trouble with type hints like this if I'm using python toolkit from an earlier version, so I prefer using the typing library for specifically typed list and dict hints